consoles/sshIucvconn.pm

```diff
@@ -35,8 +35,9 @@ sub activate {
     my ($self) = @_;
 
     my $hostname = $self->{args}->{hostname};
+    print "SSH to $hostname\n";
     my $chan = $self->backend->start_ssh_serial(hostname => $hostname, password => $self->{args}->{password}, username => 'root');
-    $chan->exec("iucvconn $hostname hvc0");
+    $chan->exec("iucvconn LINUX156 lnxhvc0");
```

Comment:
to run this in a while(1) is not working properly,  needs further investigation
still TODO: use hostname as a variable and not hardcoded